test/neura/ctrl/branch_for.mlir

```diff
@@ -74,128 +77,197 @@ func.func @loop_test() -> f32 {
 // CHECK-NEXT:   "neura.return"(%10) : (!neura.data<f32, i1>) -> ()
 // CHECK-NEXT: }
 
-// CTRL2DATA:      func.func @loop_test() -> f32 attributes {accelerator = "neura"} {
+// CTRL2DATA:   func.func @loop_test() -> f32 attributes {accelerator = "neura"} {
```

Comment:
the additional `space` before `func.func` is to make it aligned with following `-NEXT: `.